source/includes/access-mongorestore-collections.rst

```diff
@@ -5,9 +5,12 @@ access to restore any database if the backup data does not include
 
 .. include:: /includes/fact-restore-role-system.profile.rst
 
-If running :binary:`~bin.mongorestore` with :option:`--oplogReplay <mongorestore --oplogReplay>`, the
-:authrole:`restore` role is insufficient to replay the oplog. To replay
-the oplog, create a :ref:`user-defined role <create-user-defined-role>`
+As of MongoDB 3.2.11, you can run :binary:`~bin.mongorestore` with 
+:option:`--oplogReplay <mongorestore --oplogReplay>` if you have the
+:authrole:`restore` role. 
+
```

Comment:
Can we concatenate this into a single paragraph? Same topic.

Reply:
Done and restaged.